LocalSettings.php

```diff
@@ -4389,6 +4389,15 @@
 	'wgDefaultMobileSkin' => [
 		'default' => 'minerva',
 	],
+	'wgMFNamespacesWithoutCollapsibleSections' => [
+		// See https://github.com/wikimedia/mediawiki-extensions-MobileFrontend?tab=readme-ov-file#wgmfnamespaceswithoutcollapsiblesections
+		'default' => [
```

Comment:
@pskyechology I've cherry picked in the commit from #6112 and then removed the override [as you requested](https://github.com/miraheze/mw-config/pull/6112/files#r2356921423).

Is it fine to have replicated the defaults in two different places? Should I include a comment to point developers to the other place in case they get updated?

Reply:
It is the intended way and is documented in [CONTRIBUTING.md](https://github.com/miraheze/mw-config/blob/main/.github/CONTRIBUTING.md)